examples/realtime_example/dart/ffi_calls.dart

```diff
@@ -3,7 +3,9 @@ import 'dart:ffi';
 import 'drawable.dart';
 
 class WorkFfiCalls {
+  // On Linux this search fo Function in libdart_dll why on Windows it Works?
```

Comment:
`DynamicLibrary.process()` should pull in the full global symbol table all statically and dynamically linked libraries. It's posslbe that either this isn't working properly on Linux? But I'll try to double check. Either way we should likely figure it out and update the comment to be clearer.

Reply:
I've fixed FFI resolution for the realtime_example on Linux. The issue is that symbols in executables are not exported by default, so Dart can't find them. You need to add `-export-dynamic` as a linker flag for it to work. See [this commit](https://github.com/fuzzybinary/dart_shared_libray/commit/208d32ad2feb886df824348ae9adf2a3a2b08f66)